Amazon.IonObjectMapper/Attributes.cs

```diff
@@ -20,6 +20,8 @@ public class IonDoNotAnnotateType : Attribute
 
     public class IonSerializerAttribute : Attribute
     {
+        public Type Factory { get; init; }
```

Comment:
Can you please add docs to this?

Reply:
Feel free to use/augment the comments I had in https://github.com/amzn/ion-object-mapper-dotnet/pull/20/files .

Reply:
Added docs. Thanks for Justin's comments!